### PR TITLE
chore(*): change world name to imports and add package version

### PR DESCRIPTION
**Pull request by Mossaka**
* [`wit/world.wit`](diffhunk://#diff-9fe85626ec52e2a262e2361f5af2ffb65a7bb222c56f7d7029d1c70613da96a1L1-R3): Added a package version to`wasi:blobstore` package with version `0.1.0` and changed world name to imports

### CODE REVIEW

File: wit/blobstore.wit [hunk 1]
```diff
@@ -1,5 +1,3 @@
-package wasi:blobstore;
```
Comment (thomastaylor312) [inline wit/blobstore.wit:1]: Shouldn't all of these be `package wasi:blobstore@0.1.0` and not removed?

Comment (Mossaka) [reply wit/blobstore.wit:1]: I think one declaration of package is sufficient, no?

Comment (thomastaylor312) [reply wit/blobstore.wit:1]: I thought it was per file, but then I ran into other examples where it wasn't the case.

Comment (Mossaka) [reply wit/blobstore.wit:1]: Cool it seems like we are aligned here. Could you please give it another review and so I can merge it once it's approved.

**Review by thomastaylor312 — APPROVED** [review]

**Pull request merged**